pkgs/servers/x11/xorg/generate-expr-from-tarballs.pl

```diff
@@ -192,6 +192,7 @@
     if ($isFont) {
         push @requires, "fontutil";
         push @{$extraAttrs{$pkg}}, "configureFlags = [ \"--with-fontrootdir=\$(out)/lib/X11/fonts\" ];";
+        push @{$extraAttrs{$pkg}}, "postPatch = ''substituteInPlace configure --replace 'MAPFILES_PATH=`pkg-config' 'MAPFILES_PATH=`\$PKG_CONFIG' '';";
```

Comment:
Can we make this multiline easiy or does this break the script?

Reply:
it can be made multiline but it'll look worse indenting wise